MapboxDirections/MBComponent.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+/**
+The component protocol that comprises what the instruction banner should display.
+ */
+@objc(MBComponent)
+public protocol Component: class, NSSecureCoding {
+    /**
+     The plain text representation of this component.
+     */
+    @objc var text: String? { get }
```

Comment:
This property only makes sense for VisualInstructionComponent, not for LaneIndicationComponent, so remove it from this protocol (but keep it as part of VisualInstructionComponent).